steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml

```diff
@@ -0,0 +1,100 @@
+title: Fortify on Demand Mobile Assessment
+summary: |
+  Submits a Fortify on Demand Mobile Assessment
+description: |
+  Submits a Fortify on Demand Mobile Assessment
+website: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment
+source_code_url: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment
+support_url: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment/issues
+published_at: 2021-09-21T12:25:57.516922195+08:00
+source:
+  git: https://github.com/jeremy-chua/bitrise-step-fortify-on-demand-mobile-assessment.git
+  commit: 048ab9970b312fd5a60133c7de019b9b7ed19282
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
```

Comment:
Why not mobile multi-platform types like flutter, ionic etc?

Reply:
currently tested with android and ios.
others might be added in future releases